data_source.tf

```diff
@@ -0,0 +1,23 @@
+provider "oneview" {
+  ov_username   = var.username
+  ov_password   = var.password
+  ov_endpoint   = var.endpoint
+  ov_sslverify  = var.ssl_enabled
+  ov_apiversion = 2600
```

Comment:
api version is 2800 for 6.10

Reply:
we haven't release for 6.10 yet, hence keeping it to 2600 (OV6.0)